R/tm_variable_browser.R

```diff
@@ -104,6 +112,7 @@ ui_variable_browser <- function(id,
 
   shiny::tagList(
     include_css_files("custom"),
+    shinyjs::useShinyjs(),
```

Comment:
So no harm putting this here but it is called [here](https://github.com/insightsengineering/teal/blob/eb373dfdfb06aaabc97a10b7c5c7ce082d6429e3/R/include_css_js.R#L80) from [here](https://github.com/insightsengineering/teal/blob/79678e78267098d1a0377f0b6b9cd8bba1cea988/R/module_teal.R#L95)

Reply:
Ah, I was wondering whether `shinyjs` is added to all apps by default.